homeassistant/components/bme280/manifest.json

```diff
@@ -2,7 +2,11 @@
   "domain": "bme280",
   "name": "Bosch BME280 Environmental Sensor",
   "documentation": "https://www.home-assistant.io/integrations/bme280",
-  "requirements": ["i2csense==0.0.4", "smbus-cffi==0.5.1"],
+  "requirements": [
+    "i2csense==0.0.4",
+    "smbus-cffi==0.5.1",
+    "bme280spi==0.2.0"
```

Comment:
Do we need to comment this requirement in `gen_requirements_all.py` as is done for the other requirements here?

https://github.com/home-assistant/core/blob/550a6f159e72fbf143a3de781678525aab779ea0/script/gen_requirements_all.py#L28

https://github.com/home-assistant/core/blob/550a6f159e72fbf143a3de781678525aab779ea0/script/gen_requirements_all.py#L41

We do this for requirements that can't be installed on all architectures by default.

Reply:
bme280spi depends on spidev, and I do not guarantee that it will be installed on all architectures. Therefore - yes, I will add there bme280spi.